Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml

```diff
@@ -465,14 +465,14 @@
                                             <VisualStateGroup x:Name="CommonStates">
                                                 <VisualState x:Name="Normal">
                                                     <VisualState.Setters>
-                                                        <Setter Target="SelectedTabViewItemShadow.Visibility" Value="Visible" />
-                                                        <Setter Target="TabContainer.BorderThickness" Value="0" />
                                                         <Setter Target="SelectedTabViewItemShadow.Visibility" Value="Collapsed" />
-                                                        <Setter Target="TabContainer.Background" Value="{StaticResource SolidBackgroundFillColorSecondary}" />
-                                                        <Setter Target="ContentPresenter.Foreground" Value="{ThemeResource TabViewItemHeaderForegroundPressed}" />
-                                                        <Setter Target="IconControl.Foreground" Value="{ThemeResource TabViewItemIconForegroundPressed}" />
-                                                        <Setter Target="CloseButton.Background" Value="{ThemeResource TabViewItemHeaderPressedCloseButtonBackground}" />
-                                                        <Setter Target="CloseButton.Foreground" Value="{ThemeResource TabViewItemHeaderPressedCloseButtonForeground}" />
+                                                        <Setter Target="TabContainer.Background" Value="{ThemeResource SolidBackgroundFillColorSecondary}" />
+                                                        <Setter Target="TabContainer.BorderThickness" Value="0" />
```

Comment:
We should always keep the border thickness the same, but change the color to transparent rather than adjust the thickness to zero.

Reply:
@duke7553 The border isn't affected in this pull request, we can make that change in a separate PR.